include/ram_pwrdn.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file
+ * @brief RAM section power down header.
+ */
+
+#ifndef RAM_PWRDN_H__
+#define RAM_PWRDN_H__
+
+/**
+ * @defgroup ram_pwrdn RAM Power Down
```

Comment:
Spotted in .rst, but I see it's also an issue in other files in this PR: lots of capitalization inconsistencies -- RAM, ram, api, API, ID, id, and so on. Please be consistent.

Reply:
Fixed.